### PR TITLE
:memo: Corrects spelling mistake and makes minor copy changes

### PR DESCRIPTION
**Pull request by ApocalypticFish**
Relatively minor copy changes and spelling mistake correction.

### CODE REVIEW

File: Toggl.Foundation/Resources.resx [hunk 1]
```diff
@@ -297,10 +297,10 @@ categorize your tracked time.</value>
         <value>Help</value>
     </data>
     <data name="WorkspaceChangedAlertTitle">
-        <value>You've changed the workspace.</value>
+        <value>You've changed workspace.</value>
```
Comment (amulware) [inline Toggl.Foundation/Resources.resx]: Why? Where is this used?

File: Toggl.Foundation/Resources.resx [hunk 2]
```diff
@@ -297,10 +297,10 @@ categorize your tracked time.</value>
...
     </data>
     <data name="WorkspaceChangedAlertMessage">
-        <value>Your project and time entry will be allocated under the selected workspace.</value>
+        <value>Your project and time entry will be allocated to the selected workspace.</value>
```
Comment (amulware) [inline Toggl.Foundation/Resources.resx]: Why?

File: Toggl.Foundation/Resources.resx [hunk 3]
```diff
@@ -319,7 +319,7 @@ categorize your tracked time.</value>
 account is not set up</value>
     </data>
     <data name="NoEmailErrorMessage">
-        <value>Go to “Settings > Mail > Add Account” to set up an email account or send us your feedback to support@toggl.com</value>
+        <value>Go to “Settings > Mail > Add Account” to set up an email account or send your feedback to support@toggl.com</value>
```
Comment (amulware) [inline Toggl.Foundation/Resources.resx]: Why?

File: Toggl.Foundation/Resources.resx (hunk 2 above)
Comment (ApocalypticFish) [reply Toggl.Foundation/Resources.resx]: Under doesn't seem correct in this circumstance. Generally, things are allocated to projects, have tags assigned. It's a minor grammatical change but under doesn't seem correct here. They are attributed to that workspace not under it.

File: Toggl.Foundation/Resources.resx (hunk 1 above)
Comment (ApocalypticFish) [reply Toggl.Foundation/Resources.resx]: The workspace doesn't seem grammatically correct here, the workspace is where you're working. If you had changed between offices you would say you have changed office or have switched offices not you have changed the office as this suggests that you have made changes to the office or the workspace. It's not possible to make changes to the workspace from the app so this isn't the case.

File: Toggl.Foundation/Resources.resx (hunk 3 above)
Comment (ApocalypticFish) [reply Toggl.Foundation/Resources.resx]: We don't need to be identified again. It is implied that the feedback is sent to us. We don't need to further identify this by explaining that they are sending their feedback to "us". While technically still a correct and viable sentence, in my opinion, it seems more like a non-native English sentence than one a native speaker would use or is perhaps more professional seeming. Whether this is down to attempting to use the minimal amount of words/characters to convey a message or another reason I can't tell you.

Comment (amulware) [reply Toggl.Foundation/Resources.resx]: It feels a bit impersonal to me. 'send your feedback' who knows who gets it, or if anybody is gonna read it. That being said, I don't think this sentence is used anywhere anymore, and could actually be removed. But I'll let someone else check that and figure it out.

Comment (ApocalypticFish) [reply Toggl.Foundation/Resources.resx]: Okay what about "send your feedback to us at support@toggl.com" 
or "send your feedback to us support@toggl.com"

Comment (ApocalypticFish) [reply Toggl.Foundation/Resources.resx]: @amulware what do you think? I can make the changes and resubmit for review?

**Pull request merged**